src/main/java/buchen/bananagrams/Player.java

```diff
@@ -0,0 +1,62 @@
+package buchen.bananagrams;
+
+import java.util.Random;
+
+public class Player {
+
+    private final int ALPHABET_SIZE = 26;
+    private final char FIRST_CHAR = 'A';
```

Comment:
Alphabet_Size and FIRST_CHAR should be public static final

Reply:
I thought that we are NEVER supposed to use static variables?